aspects/request_timeout_handler_test.go

```diff
@@ -0,0 +1,57 @@
+package aspects
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"github.com/prebid/prebid-server/config"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var reqTimeInQueueHeaderName = "X-Ngx-Request-Time"
+var reqTimeoutHeaderName = "X-Ngx-Request-Timeout"
```

Comment:
These might be better declared `const` to make their usage clear

Reply:
Agree, fixed.